.github/workflows/commit.yml

```diff
@@ -93,7 +103,9 @@ jobs:
       - name: deploy ${{ matrix.app }}
         uses: cloud-gov/cg-cli-tools@main
         with:
-          command: cf push ${{ matrix.app }} --vars-file vars.development.yml --strategy rolling
+          command: >
+            cf push ${{ matrix.app }} --vars-file \
+            vars.development.yml --strategy rolling
```

Comment:
It's very cool that it does this for us.

Reply:
Actually good catch, I didn't need the `\` there, command works differently. See previously working version [here](https://github.com/GSA/catalog.data.gov/blob/main/.github/workflows/db-solr-sync.yml#L40-L43)

Reply:
I believe the behavior of `>` in yaml can let us break some of the other long commands into their own lines too, like in those couple that you had to disable the length linting. `>` should just append each following line without adding a newline char, right?

Reply:
Possibly. Didn't have time to look into the details, just using it for what I know would work (in the cloud.gov start command field)...